firebase-perf/src/main/java/com/google/firebase/perf/provider/FirebasePerfProvider.java

```diff
@@ -61,13 +61,11 @@ public void attachInfo(Context context, ProviderInfo info) {
     ConfigResolver configResolver = ConfigResolver.getInstance();
     configResolver.setContentProviderContext(getContext());
```

Comment:
We don't even need to initialize this class if we are not using the flag value atm.

Reply:
The goal is to kick off the background thread as soon as possible. If we lazy call it, it will not be available when we need to use a flag.